package.json

```diff
@@ -77,6 +77,21 @@
     "resolve": "^1.6.0"
   },
   "devDependencies": {
+    "@babel/helper-module-imports": "^7.0.0",
```

Comment:
yarn did this on its own.

Reply:
yeah, I forgot to reorder the packages... 😞